app/src/main/kotlin/com/simplemobiletools/flashlight/activities/MainActivity.kt

```diff
@@ -146,11 +155,13 @@ class MainActivity : SimpleActivity() {
     }
 
     private fun launchSettings() {
+        reTurnFlashlightOn = false
         startActivity(Intent(applicationContext, SettingsActivity::class.java))
     }
 
     private fun launchAbout() {
-        val licenses = LICENSE_EVENT_BUS
```

Comment:
There is two lines with same variable name val licenses = LICENSE_EVENT_BUS and val licenses = LICENSE_OTTO. Line val licenses = LICENSE_EVENT_BUS was not here previously I don't know which one to remove so i removed this line please look at this.

Reply:
keep EVENT_BUS only, that is apparently the newer one

Reply:
> keep EVENT_BUS only, that is apparently the newer one

Done